web/cypress/e2e/plugin/plugin-schema.cy.js

```diff
@@ -75,10 +75,6 @@ describe('Plugin Schema Test', () => {
             } else {
               cy.log(`${name} not a global plugin, skipping`);
             }
-
-            if (cases.length === i + 1) {
-              cy.reload(true);
-            }
```

Comment:
Frequent refreshing is the root cause of plugin-schema.cy.js' error. In fact, the logic of the test case is completely correct. It is the constant aspiration of **umi.js**